src/main/java/seedu/address/logic/parser/InsertCommandParser.java

```diff
@@ -33,6 +33,8 @@
 
 /**
  * Parses input arguments and creates a new InsertCommand object
+ * @author Amoscheong97
```

Comment:
I think the @author right you got to put above the javadoc or below, you can have a look at either mine or jeremy's classs you need to use the 
`//@@author Amoscheong97` something like this